CWE/events/soviet_union - Post Brezhnev stuff.txt

```diff
@@ -1755,7 +1755,7 @@ country_event = { # Legislative Policy
 		
 		country_event = 9001118
 		political_reform = regulated_opinion
-		political_reform = net_neutrality
+		#political_reform = net_neutrality professional_service
```

Comment:
I can't seem to edit this in the PR, could you help me add political_reform = professional_service here?

Reply:
Sure. I just commented it out because I hadn't tested the event chains with the fixes